### ISSUE TITLE
Auto save when starting debug

### ISSUE THREAD

**Issue by neclepsio** [lloiser/go-debug#86]
I couldn't find any option to automatically save files before starting the debug. If there is no such option, it would be nice to have.

**Comment by lloiser** [comment]
There is none. All files or just go?

**Comment by neclepsio** [comment]
All files. That's the way almost every IDE I used works.

**Comment by lloiser** [comment]
Added a new setting with v1.2.0.

**Comment by neclepsio** [comment]
Thank you!
